collective/jsonmigrator/blueprints/properties.py

```diff
@@ -68,9 +68,9 @@ def __iter__(self):
 
                 try:
                     if obj.hasProperty(pid):
-                        obj._updateProperty(pid, pvalue)
+                        obj._updateProperty(pid, safe_unicode(pvalue).encode('utf-8'))
```

Comment:
this change converts to string always, are you sure we want to do that?

Reply:
@hvelarde no! big mistake :)